resources/views/cart/item.blade.php

```diff
@@ -28,6 +28,18 @@ class="mx-auto"
                         </template>
                     </div>
                     @include('rapidez::cart.item.remove')
+                    <div v-if="item.qty_backordered" class="flex gap-2">
+                        <x-heroicon-o-exclamation-circle class="mt-px w-5" />
+                        <span>
+                            <template v-if="item.qty_backordered > item.qty">
```

Comment:
I'm not sure of the format qty_backordered comes in so i could be wrong.
Wouldn't it be impossible for the qty_backordered to be higher than the qty ordered?